app/models/concerns/interval.rb

```diff
@@ -63,6 +63,10 @@ def valid_date_order?
     started_on <= finished_on
   end
 
+  def leaving?
+    finished_on.present? && finished_on >= Time.zone.today
```

Comment:
I notice that we have the method `leaving?` defined in both the induction record and in teacher_status so in different contexts. I think this could be confusing / potentially lead to bugs. Perhaps we could rename the method here to `leaving_today_or_in_future ` or similar which might be more descriptive